internal/services/servicebus/servicebus_queue_data_source_test.go

```diff
@@ -40,10 +42,18 @@ func TestAccDataSourceServiceBusQueue_basic(t *testing.T) {
 	})
 }
 
+// TODO remove test in 5.0
 func TestAccDataSourceServiceBusQueue_withNamespaceName(t *testing.T) {
+	if features.FivePointOh() {
+		t.Skipf("Skipping test for 5.0")
+	}
+
 	data := acceptance.BuildTestData(t, "data.azurerm_servicebus_queue", "test")
 	r := ServiceBusQueueDataSource{}
 
+	fmt.Println(r.withNamespaceName(data))
+	t.Skip()
```

Comment:
Does this need to be here or was it just local testing?

Reply:
Ooops, good catch! Thanks, I didn't mean to commit that